docker/gogs/app.ini

```diff
@@ -0,0 +1,54 @@
+BRAND_NAME = Conduit Suite Git Repos
+RUN_USER   = git
+RUN_MODE   = prod
+
+[database]
+TYPE     = sqlite3
+HOST     = 127.0.0.1:5432
+NAME     = gogs
+SCHEMA   = public
+USER     = gogs
+PASSWORD = 
+SSL_MODE = disable
+PATH     = /data/gogs.db
+
+[repository]
+ROOT           = /data/git/gogs-repositories
+DEFAULT_BRANCH = main
+
+[server]
+DOMAIN           = localhost
+HTTP_PORT        = 80
```

Comment:
Would be nice to have SSL for new services (this is already done on the new lando). Not sure how much work it would be here, but we can punt that to a future iteration.

Reply:
Agreed. Ideally, I'd rather have simple HTTP on the services themselves, and terminate SSL in the proxy on behalf of the applications. I have a test branch to do this wholesale for suite, but it's nowhere near functional yet.